jni/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: jni
 description: Library to access JNI from dart and flutter
-version: 0.4.0
+version: 0.5.0-dev
```

Comment:
`0.5.0-dev.0` is more common in the Dart eco system I believe.

It enables one to do dev releases and then bump the version number if need be.